score-vmware/src/main/java/io/cloudslang/content/vmware/services/GuestService.java

```diff
@@ -0,0 +1,97 @@
+package io.cloudslang.content.vmware.services;
+
+import com.vmware.vim25.CustomizationSpec;
+import com.vmware.vim25.ManagedObjectReference;
+import io.cloudslang.content.vmware.connection.ConnectionResources;
+import io.cloudslang.content.vmware.constants.Outputs;
+import io.cloudslang.content.vmware.entities.GuestInputs;
+import io.cloudslang.content.vmware.entities.VmInputs;
+import io.cloudslang.content.vmware.entities.VmParameter;
+import io.cloudslang.content.vmware.entities.http.HttpInputs;
+import io.cloudslang.content.vmware.services.helpers.MorObjectHandler;
+import io.cloudslang.content.vmware.services.helpers.ResponseHelper;
+import io.cloudslang.content.vmware.services.utils.GuestConfigSpecs;
+import io.cloudslang.content.vmware.utils.ResponseUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Mihai Tusa.
+ * 3/21/2016.
+ */
+public class GuestService {
+    /**
+     * Method used to connect to specified data center and customize the windows OS based virtual machine identified
+     * by the inputs provided.
+     *
+     * @param httpInputs  Object that has all the inputs necessary to made a connection to data center
+     * @param vmInputs    Object that has all the specific inputs necessary to identify the targeted virtual machine
+     * @param guestInputs Object that has all specific inputs necessary to customize specified virtual machine
+     * @return Map with String as key and value that contains returnCode of the operation, success message with task id
+     * of the execution or failure message and the exception if there is one
+     * @throws Exception
+     */
+    public Map<String, String> customizeWinVM(HttpInputs httpInputs, VmInputs vmInputs, GuestInputs guestInputs)
```

Comment:
refactor this and customizeLinVM in at least 3 separate methods:

1) getVmMor()
2) getCustomizationSpec
3) customizeVMTask